client/asset/lcd/sendtx.go

```diff
@@ -0,0 +1,100 @@
+package lcd
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/irisnet/irishub/app/v1/asset"
+	"github.com/irisnet/irishub/client/context"
+	"github.com/irisnet/irishub/client/utils"
+	"github.com/irisnet/irishub/codec"
+	sdk "github.com/irisnet/irishub/types"
+)
+
+func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec) {
+	// create a gateway
+	r.HandleFunc(
+		"/asset/gateways",
+		createGatewayHandlerFn(cdc, cliCtx),
+	).Methods("POST")
+
+	// edit a gateway
+	r.HandleFunc(
+		"/asset/gateways/{moniker}/edit",
+		editGatewayHandlerFn(cdc, cliCtx),
+	).Methods("POST")
+}
```

Comment:
Edit gateway API should be `PUT /asset/gateways/{moniker}`

Reply:
Yes